install_config/aggregate_logging.adoc

```diff
@@ -363,6 +363,59 @@ variable allows the EFK to watch the specified audit log file or the
 default `/var/log/audit.log` file, collects audit information for the container
 engine for the platform, then puts it into Kibana.
 
+|`openshift_logging_fluentd_merge_json_log`
+|Set to `true` to enable processing of JSON logs embedded in the `log` or `MESSAGE` 
+field of the record.  The default is `true`.
+
+|`openshift_logging_fluentd_extra_keep_fields`
+|Specify a comma-separated list of fields that you do not want to be altered 
+when processing the extra fields generated when using 
+`openshift_logging_fluentd_merge_json_log`. Otherwise, Fluentd processes 
+the fields according to the other undefined field settings below.  The default is empty.
+
+|`openshift_logging_fluentd_keep_empty_fields`
+| Specify a list of comma-delimited fields to keep as empty fields when using 
+`openshift_logging_fluentd_merge_json_log`. By default, Fluentd removes fields 
+with empty values from the record, except for the `message` field. 
+
+|`openshift_logging_fluentd_use_undefined`
+|Set to `true` to move fields generated by `openshift_logging_fluentd_merge_json_log` 
+into a sub-field named by the `openshift_logging_fluentd_undefined_name` parameter. 
+By default, Fluentd keeps these at the top-level of the record, 
+which can lead to Elasticsearch conflicts and schema errors.  
+
+|`openshift_logging_fluentd_undefined_name`
+|Specify the name of the field to move undefined fields into when using 
+`openshift_logging_fluentd_use_undefined`. The default is `undefined`.
+
+|`openshift_logging_fluentd_undefined_to_string`
+|Set to `true` to convert all undefined field values into their JSON string 
+representation when using `openshift_logging_fluentd_merge_json_log`. The default is `false`.
+
+|`openshift_logging_fluentd_undefined_dot_replace_char`
+|Specify a character to replace any `.` characters in a field name, such as `_`
+when using `openshift_logging_fluentd_merge_json_log`. Undefined fields with a `.` 
+character in the name causes problems with Elasticsearch. The default is `UNUSED` 
+which means `.` in the field name is preserved.
+
+|`openshift_logging_fluentd_undefined_max_num_fields`
+|Specify a limit to the number of undefined fields when using 
+`openshift_logging_fluentd_merge_json_log`. Logs can contain hundreds of undefined fields, 
+which causes problems with Elasticsearch.  If there are more than the specified 
+number of fields, the fields will be converted into a JSON hash string and 
+stored in the `openshift_logging_fluentd_undefined_name` field.  
+The default value is `-1` which means an unlimited number of fields.
+
+|`openshift_logging_fluentd_use_multiline_json`
+|Set to true to force Fluentd to reconstruct any split log lines into a single line. 
```

Comment:
Mark up `true`.

Also the period at the end of this line is in the middle of a sentence when rendered.